packages/frontend/web/components/Outbrain.tsx

```diff
@@ -3,14 +3,49 @@
 import React from 'react';
 import { Container } from '@guardian/guui';
 
-export const shouldDisplayOutbrain = (config: ConfigType): boolean => {
+interface OutbrainSelectors {
+    widget: string;
+    container: string;
+}
+
+const shouldDisplayOutbrain = (config: ConfigType): boolean => {
+    return true;
     return config.switches.outbrainDCRTest;
 };
 
-export const OutbrainWidget: React.FC<{}> = ({}) => {
+type OutbrainSelectorsType = 'outbrain' | 'merchandising' | 'nonCompliant';
+
+const outbrainSelectorsTypeMapping = {
+    outbrain: {
+        widget: '.js-outbrain',
+        container: '.js-outbrain-container',
+    },
+    merchandising: {
+        widget: '.js-container--commercial',
+        container: '.js-outbrain-container',
+    },
+    nonCompliant: {
+        widget: '.js-outbrain',
+        container: '.js-outbrain-container',
+    },
+};
+
+const getOutbrainSelectorsByType = (
+    type: OutbrainSelectorsType,
+): OutbrainSelectors => {
+    return outbrainSelectorsTypeMapping[type];
+};
+
+const getOutbrainSelectors = (): OutbrainSelectors => {
```

Comment:
this function feels potentially superfluous since it's only called once.

Reply:
It is currently present for consistency, but will be implement later.